### PR TITLE
add - added codesniffer with moodle standart

### PR DESCRIPTION
**Pull request by mantarias**
This is a tool for making sure the pull requests follow the moodle standard.

This helps everyone follow the standard better and speeds up the review process.

We added an actions script that spins up an Ubuntu machine that sets up and runs code sniffer with moodle standard.

### CODE REVIEW

**Comment by DeeKahy** [comment]
i guess we gotta fix the issues before we can merge this

File: .phpcs.xml
Comment (DeeKahy) [inline .phpcs.xml:1]: what does this file do, and why is it needed?

Comment (ZeroConfidence) [reply .phpcs.xml:1]: specifies the what ruleset to apply i'd assume but a comment or two would be nice

Comment (mantarias) [reply .phpcs.xml:1]: It does specify what ruleset to apply, it being moodle-extra
moodle-extra is an extended ruleset which includes recommended best practices.

**Comment by AlliasGhorbani** [comment]
i have no input, makes sense, looks good

**Pull request merged**